packages/orderbook/src/orderbook.ts

```diff
@@ -209,12 +209,15 @@ export class Orderbook {
    * @param {string} listingId - The listingId to fulfil.
    * @param {string} takerAddress - The address of the account fulfilling the order.
    * @param {FeeValue[]} takerFees - Taker ecosystem fees to be paid.
+   * @param {string} amountToFill - Amount of the order to fill, defaults to sell item amount.
```

Comment:
@Sam-Jeston Isn't saying `defaults to sell item amount` problematic? Because we don't have the original sell amount here. Should we instead say - `if this parameter is not provided, the entire order will be filled by default` ?